core/src/main/java/tachyon/worker/netty/NettyDataServer.java

```diff
@@ -115,7 +114,7 @@ public boolean isClosed() {
    * Creates a default {@link io.netty.bootstrap.ServerBootstrap} where the channel and groups are
    * preset. Current channel type supported are nio and epoll.
    */
-  private ServerBootstrap setupGroups(final ServerBootstrap boot, final ChannelType type) {
+  private void setupGroups(final ServerBootstrap boot, final ChannelType type) {
```

Comment:
I think the previous way was a bit clearer, although it would have caused a side effect even if we did not update the bootstrap to the return value.